src/main/scala/uk/gov/nationalarchives/notifications/decoders/TransformEngineV2Decoder.scala

```diff
@@ -39,12 +39,12 @@ object TransformEngineV2Decoder {
 
   case class ErrorParameters(`bagit-validation-error`: BagitValidationError) extends Parameters
 
-  case class NewBagitParameters(`new-bagit`: NewBagit) extends Parameters
+  case class NewBagitParameters(`bagit-available`: NewBagit) extends Parameters
```

Comment:
I think this field name should remain as `new-bagit`

Reply:
Oops..replaceAll